content/blog/2019-05-22-ramlegacy.md

```diff
@@ -0,0 +1,51 @@
+---
+slug: "ramlegacy"
+title: "ramlegacy: a package for RAM Legacy Database"
+package_version: 0.2.0
+authors:
+  - Carl Boettiger
+  - Kshitiz Gupta
+date: '2019-05-22'
+categories: blog
+topicid:
+tags:
+- Software Peer Review
+- R
+- community
+- software
+- packages
+- ramlegacy
+- marine-biology
+- fisheries
+- stock-assessment
+---
+
+## Introduction
+
+`ramlegacy` is a new R package to download, cache and read in all the different versions of the  RAM Legacy Stock Assessment Database, a public database containing time-series and biological metrics from over 300 globally-distributed stock assessments. The package accomplishes all this by:
```

Comment:
Important to say early that RAM Legacy is "commercially exploited marine populations from around the world." or paraphrase that. I don't think the post mentioned marine animals anywhere